templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -136,7 +136,7 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Python bindings packages", file="build_python_client.sh", withenv=False) | indent(6) }}
```

Comment:
I'm don't think we should still do this here.
This has moved to the build step. Either we do a download_artifact here or we let script.sh generate and install new ones.